pages/docs/core-concepts/error-handling.mdx

```diff
@@ -152,6 +154,62 @@ const program = pipe(
 )
 ```
 
+#### Mapping over the error channel
+
+Sometimes you don't want to actually handle the error, but map over it instead. For example, you might want to add some additional information to the error, or you might want to change the error type altogether.
+
+To do this, we can use `Effect.mapError`:
+
+```ts twoslash
+// @include: errors-1
+// ---cut---
+
+const program = pipe(
+  //    ^?
```

Comment:
```suggestion
const program = pipe(
//    ^?
```

Reply:
Prettier will auto format this comment back to where it was - it aligns the comment to the code underneath it as far as I can tell.